.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## Bug Report or Feature Request (mark with an `x`)
+```
+- [ ] bug report -> please search issues before submitting
+- [ ] feature request
+```
+
+## Versions.
+```
+browser:
+browser version: 
+operating system: 
+```
+
+
+## Repro steps.
+<!--Simple steps to reproduce this bug.-->
+
+## Desired functionality.
+<!--What would like to see implemented?
+What is the usecase?-->
+
+
+### Mention any other details that might be useful.
+<!--include a gif, video or other assets that can help to fix this bug.-->
```

Comment:
Here a link to a repository with code that doesn't work would generally be more useful.